app-utils/tio/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=tio
+PKGDES="a simple serial tool"
```

Comment:
A serial device I/O tool for connecting to TTY devices

Reply:
changed